install_config/topics/configuring_azure_load_balancer.adoc

```diff
@@ -0,0 +1,124 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-load-balancer_{context}']
+= Using the Microsoft Azure external load balancer as a service
+
+{product-title} can leverage the Microsoft Azure load balancer by
+exposing services externally using a `LoadBalancer` service. {product-title}
+creates the load balancer in Microsoft Azure and creates the proper firewall
+rules.
+
+[IMPORTANT]
+====
+Currently, a bug causes extra variables to be included in the Microsoft Azure
+cloud provider when enabling external load balancers. See the following for more
+information:
+
```

Comment:
@bfallonf Is _Microsoft Azure cloud provider_ the same as _Microsoft Azure_?

Reply:
I think you're asking why cloud provider is there. Clarified.